lib/deliver/commands_generator.rb

```diff
@@ -36,7 +36,8 @@ def run
         c.action do |args, options|
           options = FastlaneCore::Configuration.create(Deliver::Options.available_options, options.__hash__)
           loaded = options.load_configuration_file("Deliverfile")
-          loaded = true if options[:description] || options[:ipa] # do we have *anything* here?
+          options.verify_conflicts
```

Comment:
Do you think there is a good way to change this so that you don't have to call this method manually? Maybe we could check it whenever we access a value? On the other hand I can see how this might cause problems when setting values on the run time.
I'm okay with leaving it like this, just want to your thought on this